package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "nethermind-creation-update-disable-detection-bot",
+  "displayName": "Nethermind Creation, Update, Disable Detection Bot",
+  "version": "0.0.1",
+  "description": "Bot that detects Nethermind's bot interactions to the Forta Agent Registry on Polygon.",
+  "longDescription": "This Forta bot monitors specific function calls made by the known Nethermind `Deployer address`. It watches transactions for `Create Agent`, `Update Agent`, and `Disable Agent` function calls directed to the `Forta Agent Registry` on Polygon.",
+  "repository": "https://github.com/Nailujj/Forta_bot_deployment",
+  "licenseUrl": "https://github.com/forta-network/forta-bot-sdk/blob/master/starter-project/LICENSE.md",
+  "promoUrl": "https://forta.org",
+  "chainIds": [137],
```

```suggestion
  "chainIds": [
      137
  ],
```
Small detail, but this is how we typically have the `chainIds` in the `package.json`.